addon/src/services/page-title.ts

```diff
@@ -7,6 +11,15 @@ import RouterService from '@ember/routing/router-service';
 import SimpleDomDocument from '@simple-dom/document';
 import type Owner from '@ember/owner';
 
+type ownerImport = { getOwner: (context: unknown) => Owner | undefined };
+let getOwner: ownerImport['getOwner'];
```

Comment:
@NullVoxPopuli fyi -- if you wanted to avoid using `any` in ember-resources, you can do something like this.

Reply:
thanks!